### PR TITLE
CIF-2612 - cloudManagerTarget config for examples deployment

### PR DESCRIPTION
**Pull request by mhaack**
## Description

Fix `cloudManagerTarget` config for CIF packages. Currently all CIF packages are ignores by CM, see below.

```
* commerce-cif-components/ui.apps/target/core-cif-components-apps-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/extensions/product-recs/content/target/core-cif-components-extensions-product-recs-content-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/examples/ui.content/target/core-cif-components-examples-content-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/examples/ui.apps/target/core-cif-components-examples-apps-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/examples/ui.config/target/core-cif-components-examples-config-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/examples/all/target/core-cif-components-examples-all-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/ui.config/target/core-cif-components-config-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/all/target/core-cif-components-all-2.4.3-SNAPSHOT.zip (skipped-content-package)
* commerce-cif-components/it/content/target/core-cif-components-it-tests-content-2.4.3-SNAPSHOT.zip (skipped-content-package)
```

To get the examples deployment working we need the `core-cif-components-all` and all example packages deployed.

## Related Issue

CIF-2612
#648

## Motivation and Context

Automatically deploy updated snapshot versions of CIF Core Components Library on aemcomponents.dev.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by mhaack** [comment]
https://github.com/adobe/aem-core-cif-components/pull/649 broke the automatic deployment since we also did exclude individual example packages from CM deployments.

**Review by Buuhuu — APPROVED** [review]

**Review by laurentiumagureanu — APPROVED** [review]

**Comment by mhaack** [comment]
Please don't merge. I will merge once the CM build is working stable.

**Pull request merged**
